src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy

```diff
@@ -919,5 +932,743 @@ enum Enum {
 '''
     }
 
```

Comment:
There are  a LOT of tests here which is great.

Perhaps its time for a new test - eg SchemaPrinterComparatorsTest